src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -1 +1,7 @@
 databaseChangeLog:
+  - include:
+      file: 001-create-table-user.yaml
+      relativeToChangelogFile: true
```

You can omit the `relativeToChangelogFile` parameter by specifying a path relative to the `resource` folder:
```suggestion
      file: db/changelog/001-create-table-user.yaml
```